### PR TITLE
[victoriametrics-k8s-stack] Upgrade dependency victoria-metrics-operator

### PR DESCRIPTION
**Pull request by Haleygo**
fix https://github.com/VictoriaMetrics/helm-charts/issues/533

### CODE REVIEW

**Comment by Haleygo** [comment]
@f41gh7 PTAL

File: charts/victoria-metrics-k8s-stack/Chart.yaml [hunk 1]
```diff
@@ -2,14 +2,14 @@ apiVersion: v2
 name: victoria-metrics-k8s-stack
 description: Kubernetes monitoring on VictoriaMetrics stack. Includes VictoriaMetrics Operator, Grafana dashboards, ServiceScrapes and VMRules
 type: application
-version: 0.14.17
-appVersion: "1.90.0"
+version: 0.14.18
```
Comment (Haleygo) [inline charts/victoria-metrics-k8s-stack/Chart.yaml:5]: I'm not sure about those version update

**Comment by Haleygo** [comment]
@k1rk @tenmozes can you guys take a look of this, thanks :)

**Review by f41gh7 — APPROVED** [review]
LGTM

File: charts/victoria-metrics-k8s-stack/Chart.yaml (hunk 1 above)
Comment (f41gh7) [reply charts/victoria-metrics-k8s-stack/Chart.yaml:5]: helm charts update a little bit weird.

1) You have to bump version (minor or major). Usually it's minor for k8s stack
2) execute make package (will bundle helm chart archieve)
3) execute make index (will add bundle into index.yaml file).

**Comment by f41gh7** [comment]
Thanks for contribution!

**Pull request merged**